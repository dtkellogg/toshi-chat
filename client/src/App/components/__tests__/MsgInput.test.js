import MsgInput from '../MsgInput'


describe('tests for MsgInput Component', () => {
  test('matches snapshot', () => {
   const wrapper = shallow(<MsgInput />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
})