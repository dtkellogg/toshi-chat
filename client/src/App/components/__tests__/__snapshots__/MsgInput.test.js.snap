// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for MsgInput Component matches snapshot 1`] = `
<form
  className="msg__input--container"
>
  <input
    className="input__msg"
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="message"
    rows="3"
    type="text"
    value=""
  />
  <button
    aria-label="Send Message"
    className="chat__btn"
    disabled={true}
    onClick={[Function]}
    type="submit"
  >
    <FaArrowAltCircleUp
      className="icon__arrow--up"
      name="send-msg"
      size={30}
    />
  </button>
</form>
`;
